exercises/sublist/example.rb

```diff
@@ -0,0 +1,18 @@
+
+def sublist(set_a, set_b)
+
+  return 'Equal' if set_a == set_b
+  return 'A is a subset' if subset(set_a, set_b)
+  return 'A is a superset' if subset(set_b, set_a)
+
+  'There are no subsets'
+end
+
+def subset(set_1, set_2)
```

Comment:
Just for the sake  of curiosity, I think an alternative implementation could be:

```ruby
def subset(set_1, set_2)
  return false if set_2.size < set_1.size

  size_diff = set_2.size - set_1.size

  (0..size_diff).each do |start|
    return true if set_2.slice(start, set_1.size) == set_1
  end

  false
end
```